src/snowflake/cli/_plugins/nativeapp/entities/application_package.py

```diff
@@ -109,6 +116,12 @@ class ApplicationPackageEntityModel(EntityModelBase):
         title="Path to manifest.yml",
     )
 
+    ### SPCS PoC
+    services: list[str] = Field(
+        title="List of Snowpark Container Service entity IDs to integrate into this application package",
+        default=[],
+    )
```

Comment:
Not sure there's a real problem here, but by omitting the proposed `children` field we lose the ability to order these internal dependencies